README.md

```diff
@@ -12,7 +12,11 @@ As this is an extension to the fybrik please make sure to install the [fybrik](h
 
 ## Installing the controller
 
-Version 0.5.0 of the controller is available in Fybrik 0.5.x.
+Version 0.5.0 of the data movement operator (DMO) is available in Fybrik 0.5.x.
+
+| Fybrik           | DMO
```

Comment:
is DMO a separate module of CBM and CSM ?

Reply:
Its not FybrikModule like CBM or CSM but rather the controller for BatchTransfer and StreamTransfer